### ISSUE TITLE
NOT ABLE TO GENERATE WEEKLY RANKING

### ISSUE THREAD

**Issue by nightswatch12** [jpmayer/fantasy-chrome-extension#120]
First off, this is a pretty amazing feature that you have created here...

For one of my leagues, I updated the database and when it was finished a message popped up saying something like the update has been completed.  In another league though, it just stops at "Week 5, Year 2018" and doesn't every fully finish, I feel like.  And therefore it won't let me click on the button to generate a weekly power ranking.  Any suggestions on help?

Thanks!

**Comment by bizzleburger** [comment]
Hi, thanks for opening an issue. Standard troubleshooting would be to clear the DB, close and open the extension, and reload the DB. Have you tried that?

**Comment by nightswatch12** [comment]
Yeah, I did that. I haven’t tried uninstalling the extension and installing it again...should i try that?
